tests/Microsoft.DotNet.Docker.Tests/PowerShellTests.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Polly;
+using Polly.Retry;
+using SharpCompress.Common;
+using SharpCompress.Readers;
+using Xunit;
+using Xunit.Abstractions;
+
+#nullable enable
+namespace Microsoft.DotNet.Docker.Tests
+{
+    [Trait("Category", "sdk")]
+    public class PowerShellTests : ProductImageTests
+    {
+        public PowerShellTests(ITestOutputHelper outputHelper)
+            : base(outputHelper)
+        {
+        }
+
+        protected override DotNetImageRepo ImageRepo => DotNetImageRepo.SDK;
+
+        public static IEnumerable<object[]> GetImageData()
+        {
+            return TestData.GetImageData(DotNetImageRepo.SDK)
+                .Where(imageData => !imageData.IsDistroless)
```

Comment:
Consider filtering by PowerShell support here instead of distroless-ness. That would also probably fix the failing Arm32/Arm64 Alpine Linux tests.

```suggestion
                .Where(imageData => imageData.SupportsPowerShell)
```